### PR TITLE
CLN: Remove encoding specifier

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [ ] closes #xxxx
- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/tests/plotting/common.py [hunk 1]
```diff
@@ -1,6 +1,4 @@
 #!/usr/bin/env python3
```
Comment (jbrockmendel) [inline pandas/tests/plotting/common.py]: do we need this shebang?

**Comment by jbrockmendel** [comment]
LGTM

File: pandas/tests/plotting/common.py (hunk 1 above)
Comment (MomIsBestFriend) [reply pandas/tests/plotting/common.py]: I meant to do this as follow up PR, trying to have each PR for specific thing.

**Comment by WillAyd** [comment]
What is the point of doing this?

**Comment by jbrockmendel** [comment]
> What is the point of doing this?

I'm pretty sure this is a py2 holdover

**Comment by MomIsBestFriend** [comment]
> > What is the point of doing this?
> 
> I'm pretty sure this is a py2 holdover

Correct, there is no need to specify the encoding (if it's utf8), in python3

**Comment by jreback** [comment]
thanks @MomIsBestFriend

**Pull request merged**